src/main/java/frc/robot/subsystems/TrapElvSubsystem.java

```diff
@@ -40,15 +40,15 @@
 import java.util.function.BooleanSupplier;
 
 public class TrapElvSubsystem extends SubsystemBase {
-  private final boolean isElv = false;
+  private boolean isElv = false;
```

Comment:
These finals were removed for testing?  Or is there a required reason that these are no longer finals?